### DISCUSSION TITLE
Navier-Stokes Set-up

### DISCUSSION THREAD

**Discussion by adfboyd** [idaholab/moose#17017] [Q&A Modules: Navier-Stokes] [answered]
Hello, I'm a student and new user of Moose. 

I am trying to set up a fluid simulation of flow moving past a sphere, and struggling to find where to get started (setting up BCs, objects, etc). I have watched some of the seminar, but am still struggling!

Any advice on where to look would be much appreciated, thank you in advance!

(If this is the wrong category I apologize, I'm happy to move it)

**Comment by ABallisat** [comment]
Have a look at the tests in the Navier Stokes module, the input files for tests are good examples of how to use all the various MOOSE components.

**Comment by aeslaughter** [comment]
We have a tutorial that will walk you through the basic steps of creating an application from scratch. This should help you grasp the core concepts, which you can apply to your problem.

https://mooseframework.inl.gov/getting_started/examples_and_tutorials/tutorial01_app_development/index.html

**Comment by lindsayad** [comment, marked as answer]
You are in the correct category! Is your flow incompressible or not? Do you want to use the finite element method or finite volume method?

**Reply by adfboyd** [reply]
Sorry for the late reply! We want incompressible flow and to use finite volume method if possible.

**Reply by GiudGiud** [reply]
You should have a look at the tests in
/modules/navier_stokes/test/tests/finite_volume/ins/ for examples of this. 
Note that this capability is fairly new in the framework so feel free to make PRs if you have new kernels, BCs or closures that could be useful to others.
